pkgs/tools/backup/mt-st/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "mt-st-1.3";
+
+  src = fetchurl {
+    url = "https://github.com/iustin/mt-st/releases/download/${name}/${name}.tar.gz";
+    sha256 = "b552775326a327cdcc076c431c5cbc4f4e235ac7c41aa931ad83f94cccb9f6de";
+  };
+
+  doCheck = false;
+
+  postConfigure = ''
```

Comment:
This can probably be replaced by specifying an `installFlags` attribute, as in `installFlags = [ "PREFIX=$(out)" "EXEC_PREFIX=$(out)" ]`